apps/challenges/views.py

```diff
@@ -19,6 +21,7 @@
 from .serializers import ChallengeSerializer, ChallengePhaseSerializer
 
 
+@throttle_classes([UserRateThrottle])
```

Comment:
@TroJan I have used `@throttle_classes` outside because the decorators work in "Inside out" manner. Now, if a user with 101th request(for throttling rate 100 requests/minute) comes then the request will be throttled initially, before checking for other decorators which can help us to save overhead on the server. Am I right ?